frontend/src/modules/entitydetails/components/Metadata.js

```diff
@@ -84,6 +86,55 @@ export default class Metadata extends React.Component<Props> {
         </Localized>;
     }
 
+    renderGroupComment(entity: Entity): React.Node {
+        if (!entity.group_comment) {
+            return null;
+        }
+
+        let comment = entity.group_comment;
+
+        return <Localized id='entitydetails-Metadata--group_comment' attrs={ { title: true } }>
+            <Property title='Group Comment' className='comment'>
+                <Linkify properties={ { target: '_blank', rel: 'noopener noreferrer' } }>
+                    { comment }
+                </Linkify>
+            </Property>
+        </Localized>;
+    }
+
+    state = { seeMore: false }
+
+    handleClick = () => {
+        this.setState({ seeMore: true });
+    };
+
+    renderResourceComment(entity: Entity): React.Node {
+        if (!entity.resource_comment) {
+            return null;
+        }
+
+        let comment = entity.resource_comment;
+
+        return <Localized id='entitydetails-Metadata--resource_comment' attrs={ { title: true } }>
+            <Property
+                title='Resource Comment'
+                className={(this.state.seeMore) ? 'comment' : 'comment truncate'}
+            >
+                <Linkify properties={ { target: '_blank', rel: 'noopener noreferrer' } }>
+                    { comment }
+                </Linkify>
+                <button
+                    id='entitydetails-Metadata--see-more'
+                    className={(this.state.seeMore) ? 'hide' : ''}
+                    title="See More"
```

Comment:
I wouldn't use a `title` attribute here, since it's the same as the button text. If you still want to keep it, please use single quotes.